CHANGES.md

```diff
@@ -895,7 +895,23 @@ OpenSSL 3.0
 
    *Richard Levitte*
 
- * Added several checks to X509_verify_cert() according to requirements in
+ * Added the `<-copy_extensions` option to the `req` command for use with `-x509`.
+   When given with the `copy` or `copyall` argument,
+   any extensions present in the certification request are copied to the certificate.
+
+   *David von Oheimb*
+
+ * The `x509`, `req`, and `ca` commands now make sure that certificates they
+   generate are RFC 5280 compliant by default.  This implies that for X.509
+   version 3 certs they ensure that a subjectKeyIdentifier extension is included
+   containig a hash value of the public key, and for not self-signed certs
```

Comment:
containing; ... for certs that are not self-signed an authorityKeyIdentifier ...

Reply:
I've fixed the typo and re-phrased the paragraph to hopefully simplify parsing the sentence,
which is rather complex because the "for X.509 version 3 certs they ensure" is relevant for both parts of the following "and" clause.